content/en/blog/2020/istiod/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Introducing istiod: simplifying the control plane"
+subtitle: Consolidating Istio components into ‘istiod’ simplifies mesh operability, while retaining Istio’s powerful functionality.
+description: A vision statement and roadmap for Istio in 2020.
+publishdate: 2020-03-19
+attribution: "Craig Box (Google)"
+keywords: [istiod,control plane,operator]
+---
+
+Microservices are a great pattern when they map services to disparate teams that deliver them, or when the value of independent rollout and the value of independent scale are greater than the cost of orchestration. We regularly talk to customers and teams running Istio in the real world, and they told us that none of these were the case for the Istio control plane. So, in Istio 1.5, we've changed how Istio is packaged, consolidating the control plane functionality into a single binary called **istiod**.
```

Comment:
istiod should be capitalized (Istiod), here and everywhere else it's used.

Reply:
I disagree with this, as it's the name of a binary/deployment/etc, which is generally all in lowercase, rather than the name of a feature (e.g. "Pilot") .  Is this up for debate?

Reply:
This is the current convention, used in all the other docs, but we can change it if we think it's wrong. All other docs use capitalized names when referring to a component generally, and backtick-escaped lowercase when referring specifically to the deployment. I think you are generally referring to Istiod as a component of Istio. If you are referring to the deployment then it should be `istiod`. Plain istiod (no backticks) will cause a lint spelling error.